metrics/Dockerfile

```diff
@@ -14,10 +14,15 @@ RUN go build -ldflags "$LDFLAGS" -tags netgo,osusergo -o metrics-exporter main.g
 
 FROM quay.io/ceph/ceph:v18
 
+# Install tini to reap zombie processes created by ceph/rbd commands
+USER root
+RUN dnf install -y tini && \
+    dnf clean all
```

Comment:
The ceph containers built by RHCS don't have `dnf` available. They use `microdnf` instead, and `tini` isn't an installable package. This might break when using RHCS images unless the build team has a different dockerfile for this

Reply:
As an alternative, I would suggest installing `tini-static` (tini built as a static binary) in a new builder stage and then copying `/usr/bin/tini-static` to the final container from the build stage

Reply:
Before going through the motions with tini, I'd suggest taking a harder look at whether there is a need to have a process reaper in place at all.  If the issue occurs only when the timeout expires, it would likely be easier to fix the respective code in `execCommand()`.